src/graph/validator/MutateValidator.cpp

```diff
@@ -233,6 +238,32 @@ Status InsertEdgesValidator::prepareEdges() {
     auto valsRet = SchemaUtil::toValueVec(row->values());
     NG_RETURN_IF_ERROR(valsRet);
     auto props = std::move(valsRet).value();
+
+    std::vector<Value> entirePropValues;
+    for (size_t j = 0; j < fieldNum; ++j) {
+      auto *field = schema_->field(j);
+      auto propName = entirePropNames_[j];
+      auto iter = std::find(propNames_.begin(), propNames_.end(), propName);
+      if (iter == propNames_.end()) {
+        if (field->hasDefault()) {
+          auto *defaultValue = field->defaultValue();
+          DCHECK(!!defaultValue);
+          auto v = defaultValue->eval(QueryExpressionContext()(nullptr));
+          entirePropValues.emplace_back(v);
+        } else {
+          if (!field->nullable()) {
+            return Status::SemanticError("The not null field doesn't have a default value.");
+          }
+          entirePropValues.emplace_back(Value(NullType::__NULL__));
+        }
+      } else {
+        auto v = props[std::distance(propNames_.begin(), iter)];
+        if (!field->nullable() && v.isNull()) {
+          return Status::SemanticError("The not null field cannot be null");
```

Comment:
ditto:

```
The non-nullable property `propName' could not be NULL.
```

Reply:
ok. I copied previous error messege in storage..